### ISSUE TITLE
[data request] HIGGS Data Set

### ISSUE THREAD

**Issue by ChanchalKumarMaji** [tensorflow/datasets#330]
* Name of dataset: `HIGGS Data Set`
* URL of dataset: https://archive.ics.uci.edu/ml/datasets/HIGGS
* License of dataset: <license type>
* Short description of dataset and use case(s): The data has been produced using Monte Carlo simulations. The first 21 features (columns 2-22) are kinematic properties measured by the particle detectors in the accelerator. The last seven features are functions of the first 21 features; these are high-level features derived by physicists to help discriminate between the two classes. There is an interest in using deep learning methods to obviate the need for physicists to manually develop such features. Benchmark results using Bayesian Decision Trees from a standard physics package and 5-layer neural networks are presented in the original paper. The last 500,000 examples are used as a test set.

Folks who would also like to see this dataset in `tensorflow/datasets`, please thumbs-up so the developers can know which requests to prioritize.

And if you'd like to contribute the dataset (thank you!), see our [guide to adding a dataset](https://github.com/tensorflow/datasets/blob/master/docs/add_dataset.md).

**Comment by ChanchalKumarMaji** [comment]
This dataset has been added in #331.

**Comment by cyfra** [comment]
Merged. Thanks a lot @ChanchalKumarMaji ChanchalKumarMaji

**Comment by MarkDaoust** [comment]
Thanks for adding this!

When I tried `tfds.load('higgs')` it looked like it was going to take about 5h to process!

TensorFlow can read csv records directly out of the `.gz` file:
 
https://www.tensorflow.org/tutorials/keras/overfit_and_underfit

Is there anyway use that trick to speed this up?
